website/static/js/quickProjectSearchPlugin.js

```diff
@@ -37,20 +37,19 @@ var quickSearchProject = {
                 self.nodes().push(node);
                 self.retrieveContributors(node);
             });
-            self.populateEligibleNodes(0, self.countDisplayed() - 1);
```

Comment:
What's up with these tweaks in this file—` - 1` here, below as well, also changing `<=` to `<`? What's the purpose and why is it happening on this PR instead of on https://github.com/caneruguz/osf.io/pull/17?